pkgs/tools/security/microsoft-identity-broker/default.nix

```
@@ -0,0 +1,106 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, openjdk11
+, jnr-posix
+, makeWrapper
+, openjfx17
+, zip
+, nixosTests
+}:
+stdenv.mkDerivation rec {
+  pname = "microsoft-identity-broker";
+  version = "1.4.1";
+
+  src = fetchurl {
+    url = "https://packages.microsoft.com/ubuntu/22.04/prod/pool/main/m/${pname}/${pname}_${version}_amd64.deb";
+    hash = "sha512-d7eKqB9+OUr/B1ZMwX9T9PfbBn9KpUEmJcCe7LHOoMj1fsETDIjvLmXtzijGwVLhYc1JvjbJXEWoTw0oBb9cFQ==";
+  };
+
+  patches = [ ./execstartpre.patch ];
```

Please add a comment with an explanation why it is needed.

```suggestion
  patches = [
    # Do not migrate legacy directory in ExecStartPre, it makes …
    ./execstartpre.patch
  ];
```